### ISSUE TITLE
Animation of WebImageView still occurs within a list view

### ISSUE THREAD

**Issue by charroch** [mttkay/droid-fu#1]
The animation of the appearance of the image still occurs even after loaded when WebImageView is used within a ListView. The imageSrc is set dynamically with loadImage() which loads the image accordingly but the animation still remains after the image was loaded. This is apparent when you scroll down a list and then come back to the original position. Applying the logic within the efficient adapter from the API demos or removing the animation after the image was loaded could be 2 potential solution. I will work on a solution but though about creating the issue before hand.

**Comment by mttkay** [comment]
Yes, that's because the view doesn't distinguish between an image coming from the cache or the web. I guess this is a problem specific to ListView, because it's the only view that converts (recycles) old views. Removing the animation after image loading sounds reasonable though!
